test/repl/proxysql_repl_tests/bin/proxy-repl-casc-config.bash

```
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+. constants
+
+# check if all required env vars are set
+if [ -z "${INFRA}" ]; then
+    echo "INFRA is empty, please check env vars passed to: " ${0}
+    exit 1
+elif [ -z "${RHG2}" ]; then
+    echo "RHG1 is empty, please check env vars passed to: " ${0}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)